src/SelectBox/index.tsx

```diff
@@ -115,41 +122,41 @@ const SelectBox = (props: SelectProps) => {
     )
   }
 
-  const onFocus = (event: any) => {
+  const onFocus = (event: React.FocusEvent<HTMLSelectElement>) => {
     const { onFocus } = props
     event.preventDefault()
-    onFocus && onFocus()
+    onFocus && onFocus(event)
   }
 
-  const onBlur = (event: any) => {
+  const onBlur = (event: React.FocusEvent<HTMLSelectElement>) => {
     const { onBlur } = props
     onBlur && onBlur(event)
   }
 
-  const mobileOnChange = (e: any) => {
-    const value = props.options.find((o: any) => o.value === e.target.value)
-    props.onChange(value)
+  const mobileOnChange = (e: React.ChangeEvent<HTMLSelectElement>) => {
+    const option = props.options.find((o: Option) => o.value === e.target.value)
+    props.onChange(option)
   }
 
   // @ts-ignore
   if (isBrowser && window.mobilecheck() && props.useNativeSelectOnMobile) {
     return (
       <ThemeProvider theme={theme}>
-        <>
+        <div className={props.className}>
           {props.description && (
             <StyledDescription>{props.description}</StyledDescription>
           )}
           <SelectContainer>
             <MobileSelectWrap>
               <MobileStyledSelect
                 {...props}
-                onFocus={(event: any) => onFocus(event)}
+                onFocus={onFocus}
                 onBlur={onBlur}
                 value={props.value && props.value.value}
-                onChange={e => mobileOnChange(e)}
+                onChange={mobileOnChange}
               >
                 <option value="" disabled selected></option>
-                {props.options.map((o: any) => (
+                {props.options.map(o => (
                   <option
```

Comment:
this should have `key` attribute (I would add label as key, they should be unique as a option)